### DISCUSSION TITLE
Export of attachments to encrypted vault not happening

### DISCUSSION THREAD

**Discussion by pcuttriss** [padloc/padloc#663] [Q&A] [answered]
Afternoon all

I have done a lot of digging around to try and work out what the expected behaviour is and spun up a docker instance successfully to  test attachment export.

Latest version pulled from the docker registry using the example for hosting from here: 
https://github.com/padloc/padloc/tree/main/docs/examples/hosting/docker/postgres-nginx-letsencrypt
Created an account successfully.
Login and create an item which has a single file attached.
Export to encrypted vault. File size indicates the export does NOT contain the attachment.
Delete item in online vault.
Import previous export.
Item is recreated but no attachment is in place.

Is this expected behaviour? i.e. exports do NOT include attachments

I expect an export to CSV not to include attachments...but I was hoping that export to encrypted vault would include attachments.

If this is expected behaviour is there any way to export attachments?

Associated with this is an additional problem. 
If I create an item which has a single file attached, do an export to encrypted vault, and then import that same encrypted vault (without touching or deleting the item that currently exists with an attachment), the item exists but the attachment is no longer accessible. However when looking at the docker filesystem I can see the attachment still exists in the container storage.

Thanks
Paul

**Comment by BrunoBernardino** [comment]
Thanks for reporting this @pcuttriss ! Indeed there's no standard way to export attachments (even 1Password didn't support it for a long time, though I think they now do). This would be classified as a feature request.

**Comment by pcuttriss** [comment, marked as answer]
Hi Bruno

Thanks for the response.

I have opened issue #664 Feature request: Export attachments to encrypted vault to track.

Paul C
